validator/Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "spaceapi-validator"
 version = "0.1.0"
+repository = "https://github.com/spacedirectory/validator"
 authors = ["Raphael Nestler <raphael.nestler@gmail.com>"]
+description = "Validator for Space API JSON data."
+keywords = ["spaceapi", "validator", "hackerspaces", "status", "api"]
+license = "MIT OR Apache-2.0"
```

Comment:
Hmm according to http://doc.crates.io/manifest.html#package-metadata it should be MIT/Apache-2.0